gcc/rust/backend/rust-compile-expr.cc

```diff
@@ -692,12 +692,6 @@ CompileExpr::visit (HIR::MatchExpr &expr)
 	  }
 	  break;
 
-	  case HIR::Expr::ExprType::Ident: {
```

Comment:
Is this the piece you were asking about @dafaust :D hopefully this helps you

Reply:
Yes exactly! I couldn't find a case where we would hit this and not just the `ExprType::Path` below so I started wondering :smile: